submissions/opensmt.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "OpenSMT",
+    "contributors": [
+        { "name": "Tomáš Kolárik", "website": "https://github.com/Tomaqa" }
+        ,
+        { "name": "Martin Blicha", "website": "https://github.com/blishko" }
+    ],
+    "contacts": [
+        "Tomáš Kolárik <tomas.kolarik@usi.ch>"
+        ,
+        "Martin Blicha <martin.blicha@usi.ch>"
+    ],
+    "archive": {
+        "url": "https://zenodo.org/doi/10.5281/zenodo.11371846"
```

Comment:
@Tomaqa Thanks for submitting OpenSMT to SMT-COMP! The link to the archive should point directly to the archive, not to the general Zenodo submission (we are downloading the archive automatically by a script). Could you change it in the pull request? In your case, it would be something like `"url": "https://zenodo.org/records/11371847/files/opensmt.tgz"`.

I fixed that locally and tried running the current submission on our competition infrastructure with some trivial benchmarks. Everything else seems to be working so far!

Reply:
Done. Not sure if URLs of files are guaranteed to be persistent, though.